plotly/offline/offline.py

```diff
@@ -298,3 +309,128 @@ def plot(figure_or_data,
             ])
         else:
             return plot_html
+
+
+def plot_mpl(mpl_fig, resize=False, strip_style=False,
+             verbose=False, **kwargs):
```

Comment:
no more `**kwargs`, they kill us down the line! Because we'll add a new keyword argument to `plot_mpl` in a future version but then silently fail on *this* version when the person supplies that (unsupported) keywoard argument